hack/community/csv/description.md

```diff
@@ -0,0 +1,34 @@
+    ## WARNING
```

Comment:
Format the commit message following the [guidelines](https://github.com/openshift/windows-machine-config-operator/blob/master/CONTRIBUTION.md#format-of-the-commit-message).